### PR TITLE
Adding Get-CsUserSettingsPageConfiguration

### PR DESCRIPTION
**Pull request by get-itips**
@kenwith

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsUserSettingsPageConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -LocalStore
+Retrieves the user pool information from the local replica of the Central Management store rather than from the Central Management store itself.
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsUserSettingsPageConfiguration.md]:
```suggestion
This parameter is reserved for internal Microsoft use.
```

Comment (kenwith) [reply skype/skype-ps/skype/Get-CsUserSettingsPageConfiguration.md]: @get-itips what do you think?

Comment (get-itips) [reply skype/skype-ps/skype/Get-CsUserSettingsPageConfiguration.md]: my bad, @dariomws suggested correctly to put that text based on the metadata that said this was an Online cmdlet, however it is a Server cmdlet, pushed another commit to correct it.

Comment (dariomws) [reply skype/skype-ps/skype/Get-CsUserSettingsPageConfiguration.md]: Thanks!

**Pull request merged**